app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Kindly remove the `foreign_key: 'author_id'`. Remember that the `User` has_many `Likes`, therefore the `foreign_key` will be on the `Likes` Table. 👍